src/main/java/com/google/sps/servlets/UserServlet.java

```diff
@@ -56,33 +55,32 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   /** Inserting hardcoded user data into the Datastore
   */
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException{
```

Comment:
Consider deleting the hardcoded data and replacing this with the doPost method you implemented in UserCreationServlet? I think just one servlet is enough to handle the creation of users and retrieval of user data.

Reply:
okay thats true! let me combine it together.